README-DARWIN.md

```diff
@@ -2,8 +2,8 @@ So, you want to cross-compile for darwin? Have fun.
 
 Jokes, I've done all the work for you. Simply:
 
-0. Make the `osxcross` darwin clang toolchain available on your PATH (`rewind-server` has a handy
-   script to do all the building for you)
+0. Make the `osxcross` darwin clang toolchain available on your PATH
+   (Fourier application repositories have tooling to do all the building for you).
```

Comment:
I can see the tension here between keeping the docs useful and keeping them maintainable, but it might be better to err on the side of usefulness here. Consider:

```suggestion
   (The scripts `transform/desktop/build/build-darwin-cross-compiler.sh` and
   `rewind-server/build-darwin.sh` may be useful.)
```